src/pretix/plugins/banktransfer/templates/pretixplugins/banktransfer/transaction_list.html

```diff
@@ -4,7 +4,14 @@
 {% load static %}
 <div class="table-responsive">
     {% csrf_token %}
-    <table class="table table-condensed transaction-list" data-url="{% if request.event %}{% url "plugins:banktransfer:import.action" event=request.event.slug organizer=request.organizer.slug %}{% else %}{% url "plugins:banktransfer:import.action" organizer=request.organizer.slug %}{% endif %}">
+    <table class="table table-condensed transaction-list" data-url="
+
+
+
+
+
+
+            {% if request.event %}{% url "plugins:banktransfer:import.action" event=request.event.slug organizer=request.organizer.slug %}{% else %}{% url "plugins:banktransfer:import.action" organizer=request.organizer.slug %}{% endif %}">
```

Comment:
```suggestion
    <table class="table table-condensed transaction-list" data-url="{% if request.event %}{% url "plugins:banktransfer:import.action" event=request.event.slug organizer=request.organizer.slug %}{% else %}{% url "plugins:banktransfer:import.action" organizer=request.organizer.slug %}{% endif %}">
```

Reply:
I don't know why github shows this. Looks fine in my editor...